source/ProcessManager.Orchestrations/Processes/BRS_021/ForwardMeteredData/V1/Handlers/EnqueueMeasurementsHandlerV1.cs

```diff
@@ -149,6 +149,16 @@ private List<ReceiversWithMeteredDataV1> CalculateReceiversWithMeasurements(
                     meteringPointMasterData,
                     measurements));
 
+        receiversWithMeasurements.Add(
+            new ReceiversWithMeasurements(
+                Receivers: customState.AdditionalRecipients,
+                Resolution: forwardMeteredDataInput.Resolution,
+                MeasureUnit: forwardMeteredDataInput.MeasureUnit,
+                StartDateTime: forwardMeteredDataInput.StartDateTime.ToDateTimeOffset(),
+                EndDateTime: forwardMeteredDataInput.EndDateTime.ToDateTimeOffset(),
+                GridArea: meteringPointMasterData.First().CurrentGridAreaCode.Value,
```

Comment:
```suggestion
                GridArea: meteringPointMasterData.First().CurrentGridAreaCode.Value,
```

This `First()` is ok?

Reply:
there is a local variable named `gridAreCode` consider using it